packages/dropdowns/src/elements/Menu/Items/ItemMeta.spec.tsx

```diff
@@ -33,4 +34,24 @@ describe('ItemMeta', () => {
 
     expect(getByTestId('item-meta')).toBe(ref.current);
   });
+
+  it('renders danger styling if provided', () => {
+    const { getByTestId } = render(
+      <Dropdown isOpen>
+        <Trigger>
+          <button data-test-id="trigger">Test</button>
+        </Trigger>
+        <Menu>
+          <MediaItem isDanger value="image">
+            <MediaBody>
+              Image Media Item
+              <ItemMeta data-test-id="item-meta">Meta info</ItemMeta>
+            </MediaBody>
+          </MediaItem>
+        </Menu>
+      </Dropdown>
+    );
+
+    expect(getByTestId('item-meta')).toHaveStyleRule('color', getColor('dangerHue'));
+  });
```

Comment:
This repo organizes style specific tests so that these could probably live in styled directory, e.g. `/dropdowns/src/styled/items/StyledItemMeta.ts`.